drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -102,15 +102,30 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
 
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
-        ChromeOptions chromeOptions = new ChromeOptions();
-        CapabilitiesOptionsMapper.mapCapabilities(chromeOptions, capabilities, BROWSER_CAPABILITIES);
-
         // verify binary capabilities
         if (Validate.nonEmpty(binary)) {
```

Comment:
yeah, you are right with this - that the `setChromeOptions` should only set... then leave it as it is...